STARTERKIT/composer.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "drupal/skilld-zen",
+    "description": "Skilld Zen theme fork.",
+    "type": "drupal-theme",
+    "license": "GPL-2.0",
+    "require": {
+        "oomphinc/composer-installers-extender": "^1.1",
+        "fxp/composer-asset-plugin": "~1.3",
+        "npm-asset/slick-carousel": "^1.7"
+    },
+    "extra": {
+        "installer-types": [
+            "bower-asset",
+            "npm-asset"
+        ],
+        "installer-paths": {
+            "libraries/{$name}": [
```

Comment:
Why do we need `libraries` folder and not just vendor ?

Reply:
According to our theme structure all libraries stacked in folder 'libraries'.